src/Processors/QtiV2/Out/QuestionTypes/McqMapper.php

```diff
@@ -26,11 +32,30 @@ public function convert(BaseQuestionType $questionType, $interactionIdentifier,
         // Build <choiceInteraction>
         $valueIdentifierMap = [];
         $simpleChoiceCollection = new SimpleChoiceCollection();
+        $inlineCollection = new InlineCollection();
+        $metadata = $question->get_metadata();
+
+        if (isset($metadata) && !empty($metadata->get_distractor_rationale_response_level())) {
+            $feedbackOptions = [];
+            foreach ($metadata->get_distractor_rationale_response_level() as $feed):
+                $feedbackOptions[] = $feed;
+            endforeach;
+        }
+        $i = 0;
         foreach ($question->get_options() as $index => $option) {
             /** @var mcq_options_item $option */
             $choiceContent = new FlowStaticCollection();
             foreach (QtiMarshallerUtil::unmarshallElement($option->get_label()) as $component) {
+
                 $choiceContent->attach($component);
+                // attach feedbackInline to simpleChoice
+                if (isset($feedbackOptions) && $feedbackOptions[$i] != '') {
```

Comment:
Use identity comparison `!==` to avoid type juggling issues.

Reply:
Done